### ISSUE TITLE
"No results found."

### ISSUE THREAD

**Issue by TheSnake** [leetreveil/Zune-Social-Tagger#8]
What it says on the tin. I'm in the US; I have access to the Marketplace and everything. Is this perhaps happening because the Marketplace says it's unavailable (despite it working anyway)?

**Comment by leetreveil** [comment]
can you copy and paste the error message that ZST displays? Thanks.

**Comment by TheSnake** [comment]
All it says is NO RESULTS.

**Comment by TheSnake** [comment]
Alright, here goes: The reason I looked into this software is because a few days ago, my artist bio and pictures information randomly disappeared from my phone. Re-syncing, etc. from Zune didn't help. Then, a couple days later, Zune Marketplace keeps saying it's temporarily unavailable. It's been like that for a few days. Well, I just installed Zune onto my mom's WinXP computer and...it doesn't have this problem! Marketplace is working fine, as is Zune Social Tagger. I'm extremely confused and while I didn't change any options in Zune or my firewall, I'm starting to think that perhaps it (and your software) are having trouble communicating with the Zune server. It might be worth mentioning, however, that despite the fact that this WinXP machine isn't experiencing the Marketplace problem, I'm still not getting my bios or pictures back, which brings me back to square one anyway :(
